### ISSUE TITLE
Change coveralls from comment to status

### ISSUE THREAD

**Issue by nstarman** [beartype/plum#159]
There's not much info in the comment that isn't in the status check. The old comments don't get removed after a new commit.
Status checks can be made required to merge a PR.
See https://docs.coveralls.io/app-notifications

**Comment by wesselb** [comment]
Ah, this would be really nice! I checked, but unfortunately I don't seem to have access to the settings page of the Plum Coveralls page since Plum moved to @beartype.

@leycec, I noticed that Coveralls access to the @beartype organisation is still pending:

<img width="963" alt="image" src="https://github.com/beartype/plum/assets/1444448/5c29aa6e-2cc3-4226-bf7d-6fa89accf2fa">

Would you be willing to approve that?

**Comment by leycec** [comment]
**Gah!** So sorry about that, everybody. Everything shall immediately be permitted. And... actually, I'm lost. I couldn't figure out how to do anything. Instead, I [just enabled the **Discussions** tab for Plum](https://github.com/beartype/plum/discussions) and [invited everybody to shower @wesselb with copious praise and fawning affection](https://github.com/beartype/plum/issues/159). Please now do this.

**Comment by leycec** [comment]
**Storytime.** I fell down the GitHub Settings UX labyrinth and couldn't get out. I almost gave up. My stamina was depleted. Then I valiantly performed a random walk across the heterogenous landscape of non-Euclidean madness that is the GitHub Settings UX. Finally, a shining beacon of sanity shown dimly through the haze of my pounding migraine: under "OAuth application policy," a web dev phrase that's basically meaningless to me, I found something! I found a button. I pushed a button. Sunlight filtered through. A single solitary plum fell off a nearby tree and rolled into the dim miasma of the labyrinth.

With tears in my glaucoma-addled eyes, I ate that plum. Now, so can you.

**English Translation:** *I approved Coveralls access to the @beartype organization.* :muscle:

**Comment by wesselb** [comment]
@leycec Ahh, thank you for getting to the bottom of this so quickly. :) That's super helpful! I can confirm that I have access now. 🎉

@nstarman Unfortunately, I'm having some trouble getting Coveralls to do the check. I've configured the repo in this way:

<img width="671" alt="image" src="https://github.com/beartype/plum/assets/1444448/21b79b6b-07c5-4e83-8a1a-e3e81da938b0">

[I then deleted some tests in a test PR](https://github.com/beartype/plum/pull/162), but, no matter how I configure the settings, I can't seem to get the status check to fail: it just always passes.

Any idea what could be going on?
